pkgs/top-level/python-aliases.nix

```
@@ -42,6 +42,7 @@ mapAliases ({
   class-registry = phx-class-registry; # added 2021-10-05
   ConfigArgParse = configargparse; # added 2021-03-18
   cozy = throw "cozy was removed because it was not actually https://pypi.org/project/Cozy/."; # added 2022-01-14
+  cryptography_vectors = cryptography-vectors;
```

```suggestion
  cryptography_vectors = cryptography-vectors; # Added 2022-03-23
```

So this can be automatically cleaned up using #161146

---

Nice! I didn't know there was such script!